MSAL/src/public/MSALAuthenticationSchemePop.m

```diff
@@ -0,0 +1,54 @@
+//------------------------------------------------------------------------------
+//
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//------------------------------------------------------------------------------
+
+#import "MSALAuthenticationSchemePop.h"
+#import "MSIDAuthenticationSchemePop.h"
+#import "MSALHttpMethod.h"
+
+@implementation MSALAuthenticationSchemePop
+
+- (instancetype)initWithHttpMethod:(MSALHttpMethod)httpMethod requestUrl:(NSURL *)requestUrl
+{
+    self = [super init];
+    if (self)
+    {
+        _scheme = MSALAuthSchemePop;
+        _httpMethod = httpMethod;
+        _requestUrl = requestUrl;
+        _nonce = [[NSUUID UUID] UUIDString];
```

Comment:
nonce doesn't seem to be used anywhere

Reply:
Forgot to remove this one. Its only needed for MSID layer for now. I have to check with Android to see if the developer can pass his own nonce in which case we might add it here and pass it to common core layer.